lib/getPackageDetails.js

```diff
@@ -70,6 +72,11 @@ module.exports = function getPackageDetails(
     return Promise.resolve(null);
   }
   const key = `${name}@${versionLoose}`;
+  const scope = (/(@[^\/]+)\/.*/.test(name) && /(@[^\/]+)\/.*/.exec(name)[1]) || ``;
+  var registryUrl = (scope && npmConfig[`${scope}:registry`]) || npmConfig[`registry`];
```

Comment:
`let` please

Reply:
fixed together with the remaining eslint errors